packages/platforms/mellanox/x86-64/x86-64-mlnx-msn2700/onlp/builds/src/module/src/platform_lib.c

```diff
@@ -0,0 +1,165 @@
+/************************************************************
```

Comment:
The deviceNode() functions in the (copied) platform_lib.{c,h} should be removed. 
Please use the equivalent functions already provided by the ONLP libraries for this purpose:
https://github.com/opencomputeproject/OpenNetworkLinux/blob/master/packages/base/any/onlp/src/onlplib/module/inc/onlplib/file.h